### DISCUSSION TITLE
I tried to deploy a Vite react 3d website but the homepage for the website is not where it's supposed to be

### DISCUSSION THREAD

**Discussion by Mwahhid** [community/community#109764] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I created a 3d website using Vite and react and am now trying to deploy it to github pages. I looked at some youtube tutorial and created a workflow to put my website on github pages. But when I open my website by going to https://mwahhid.github.io/My_3D_Portfolio/ I get to this incomplete page. 
![image](https://github.com/community/community/assets/78094284/c803640f-f065-4dad-a251-df784c30c1ad)

This is where my homepage is supposed to be. You can access my homepage by clicking on my initials "MM" as shown on the top left in the picture above. What do I need to change to get this homepage to open every time I open https://mwahhid.github.io/My_3D_Portfolio/ ?

The source of the github page is set to the 'gh-pages' branch.

The link to my repo: https://github.com/Mwahhid/My_3D_Portfolio/

**Comment by queenofcorgis** [comment]
Hi there - wanted to check in on this as I went to investigate and I'm not getting a blank page, but a 404 error. Curious if you made any changes since posting?

**Reply by Mwahhid** [reply]
Thank you, I was able to fix it. I just put up the page on https://mwahhid.github.io/ instead

**Comment by amankr1469** [comment]
Make sure you have added Keys and values (which you usually store in .env) while deploying your project. 
You usually get a blank screen when you don't add keys correctly or miss anyone.
